languagetool-core/src/main/java/org/languagetool/CsvToXml.java

```
@@ -0,0 +1,132 @@
+      });
+      frame.add(btnFile);
+      frame.add(text);
+      frame.add(split);
+      frame.add(btnOk);
+      frame.pack();
+      frame.setVisible(true);
+    }
+  }
+
+  public static void csvToBitext(String csvFile, String cvsSplitBy, String line) {
+
+    try (BufferedReader br = new BufferedReader(new FileReader(csvFile))) {
+      DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+      DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+
+      Document doc = docBuilder.newDocument();
+      Element rootElement = doc.createElement("example");
+      doc.appendChild(rootElement);
+
+      Element example00 = doc.createElement("example00");
+      rootElement.appendChild(example00);
+
+      Element example01 = doc.createElement("example01");
+      rootElement.appendChild(example01);
+
+      example00.setAttribute("lang", "en");
+
+      example01.setAttribute("lang", "ru");
+
+      while ((line = br.readLine()) != null) {
+
+        String[] x = line.split(cvsSplitBy);
+
+        Element example000 = doc.createElement("example000");
+        example000.appendChild(doc.createTextNode(x[4]));
+        example00.appendChild(example000);
+
+        Element example010 = doc.createElement("example010");
+        example010.appendChild(doc.createTextNode(x[5]));
+        example01.appendChild(example010);
+      }
+
+      // write the content into xml file
+      TransformerFactory transformerFactory = TransformerFactory.newInstance();
+      Transformer transformer = transformerFactory.newTransformer();
+      DOMSource source = new DOMSource(doc);
+      StreamResult result = new StreamResult(new File("file.xml"));
+
+      // Output to console for testing
+//            StreamResult result = new StreamResult(System.out);
+      transformer.transform(source, result);
+
+      System.out.println("File saved!");
+
+    } catch (IOException | ParserConfigurationException ex) {
```

Please don't ignore exceptions